src/icons/plus.jsx

```diff
@@ -0,0 +1,10 @@
+export default ({ width, height }) => {
```

Comment:
we have Add icon that you can use

Reply:
I have used Add icon.
Should I remove Play icon ?

Reply:
yes please remove Plus icon

Reply:
Done